.coveragerc

```diff
@@ -1,4 +1,7 @@
 [run]
+source =
+    src
+    test
```

Comment:
We haven't historically included `test` in the test coverage, so if it works, this may result in a decrease in overall coverage.  Looking at our `Makefile` the coverage arguments we have defined are:

For use with `coverage run`:
```
--append --source=src/pudl
```

For use with `pytest`:
```
--cov-append --cov=src/pudl --cov-report=xml
```

The command we use to generate the coverage report:
```
coverage report --sort=cover
```